lib/rules/rule-selector-property-disallowed-list/__tests__/index.mjs

```diff
@@ -89,6 +97,14 @@ testRule({
 			endLine: 1,
 			endColumn: 27,
 		},
+		{
+			code: 'a { b { width: 100% } color: red; }',
+			message: messages.rejected('a', 'color'),
+			line: 1,
+			column: 23,
+			endLine: 1,
+			endColumn: 28,
+		},
```

Comment:
[suggest] How about adding the following case?

```js
{
	code: 'b { a { color: red; } }',
	message: messages.rejected('a', 'color'),
	// ...
}
```

Besides, how about adding more *accepted* cases? E.g.

```js
	{
		code: 'a { b { color: red; } }',
	},
	{
		code: 'b { & a { color: red; } }',
	},
```